bin/create-gxa-analytics-schema.sh

```diff
@@ -27,6 +27,101 @@ curl -X POST -H 'Content-type:application/json' --data-binary '{
 
 #############################################################################################
 
+printf "\n\nDelete field is_private\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "delete-field":
+  {
+    "name": "is_private"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+printf "\n\nCreate field is_private (boolean)\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "add-field":
+  {
+    "name": "is_private",
+    "type": "boolean"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+#############################################################################################
+
+printf "\n\nDelete copy field rule bioentity_identifier -> bioentity_identifier_search\n"
```

Comment:
Please move this `delete copy field rule `snippet to before `add copy field rule` and similarly align `bioentity_identifier`, ``bioentity_identifier_search``field order as well.

Reply:
I think that there is a dependency between fields that means that copy field rule might need to be deleted before you can delete bioentity_identifier, and in turn the copy field action needs to be created after the field is copying (bioentitiy_identifier). The same with the search. This is as well the same order that it was in the source repo (I didn't change the order).

Reply:
Yes, you're right @pcm32.Thanks.